robotiq_joint_state_publisher/src/s_model_joint_states.cpp

```diff
@@ -0,0 +1,208 @@
+//  The MIT License (MIT)
```

Comment:
Would you be willing to re-license this contribution?  The accepted ROS-I licenses are BSD or Apache 2.0 (preferred).